projects/gluster.yml

```
@@ -0,0 +1,5 @@
+name: Gluster
+url: https://www.gluster.org/
+modules:
```

```suggestion
mixins:
```

These seem to be dashboards, not kubernetes objects